operations/elections.rst

```diff
@@ -0,0 +1,78 @@
+Volby do výboru
+===============
+
+Funkční období :term:`výboru <Výbor>` je dle :ref:`stanov <stanovy>` tři roky, členové spolku ze svých řad volí pět členů. To se děje na členské schůzi, kterou výbor k této příležitosti svolává.
+
+Nově zvolený výbor si potom sám volí, kdo z těchto pěti bude předseda a případně kdo bude pokladník (když se nerozhodne volit pokladníka, je jím automaticky předseda).
+
+.. note::
+    Přečíst ještě `PEP 8102 <https://www.python.org/dev/peps/pep-8102/>`__ a případně se inspirovat? Dořešit.
+
+Kdy se volí
+-----------
+
+- Předchozí volba: **8.4.2019**
+- Příští volba: **?.?.2022**, předpokládá se termín březen/duben
+- Příprava pro rok 2022 začíná: od 1. ledna
+
+Příprava
+--------
+
+- Postupuje se podle stanov, svoláváme :ref:`členskou schůzi <clenska-schuze>`.
+- Výbor na nějaké schůzi s předstihem určí přesné datum členské schůze.
+- Někdo z výboru vezme e-mailové adresy členů ze `seznamu členů <https://docs.google.com/spreadsheets/d/1n8hzBnwZ5ANkUCvwEy8rWsXlqeAAwu-5JBodT5OJx_I/edit#gid=0>`__ a pošle jim pozvánku na členskou schůzi. Pozvánka musí obsahovat místo, čas a program jednání členské schůze. Kromě toho by měl výbor schůzi oznámit i v kanálu :slack:`#pyvec-members`. (Pro inspiraci, pozvánka na členskou schůzi `EuroPython Society <https://www.europython-society.org/europython-society-general-assembly-2020/>`__.)
+- Výbor dá členům vědět, jak lze podávat nominace, jak kandidovat. Kandidovat do výboru může jakýkoliv člen Pyvce.
+- Ideálně se pokusí členy motivovat k tomu, aby kandidovali, a to komunikací s členy, ukazováním toho, jak výbor pracuje, jak vypadá běžná schůze výboru (např. zveřejněním video nahrávky), apod. Výbor může také neformálně oslovovat vhodné kandidáty a přesvědčovat je, že mají kandidovat.
+- Přihlášky se oznamují v kanálu :slack:`#pyvec-members`. (Pro inspiraci, `kandidatury do EuroPython Society <https://www.europython-society.org/list-of-eps-board-candidates-for-20192020/>`__.) Seznam kandidátů by bylo dobré zapisovat třeba sem do této stránky, ať je někde přehledný seznam. Pozor na GDPR, kontrolovat v `seznamu členů <https://docs.google.com/spreadsheets/d/1n8hzBnwZ5ANkUCvwEy8rWsXlqeAAwu-5JBodT5OJx_I/edit#gid=0>`__, zda daná osoba může být někde veřejně takto prezentovaná.
+- Neomezujeme kandidáty na ty přihlášené předem. Kandidáti se mohou navrhnout nebo nominovat jiné kandidáty klidně až v průběhu hlasování. Je ale pro hlasující lepší, když se mohou s kandidáty seznámit předem.
```

Comment:
Bude potřeba zkontrolovat jestli to umožňuje zvolené technické řešení. (U Heliosu nevím, ale nedivil bych se kdyby to nešlo.)

Reply:
Ano, přidali jsme to do poznámky v rozpracovaném https://github.com/pyvec/docs.pyvec.org/pull/262